platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -120,7 +133,6 @@ public void init() throws Exception {
         return migrationWarnings;
     }
 
-    @Override
```

Comment:
`@Override` works when you implement an interface method as well so we should keep it.